node/res/chain_specs/litmus-plain.json

```diff
@@ -169,10 +169,7 @@
       "aura": {
         "authorities": []
       },
-      "auraExt": null,
-      "sudo": {
-        "key": "5ECCQzRn2nUqDesngcoi7FxLhJBXGSnoEmXiMDZXCj94chRP"
-      }
+      "auraExt": null
```

Comment:
Sure, we will keep it; Just wondering any specific reason to keep it?

Reply:
litmus-plain.json is a 1-to-1 mapping to litmus.json which is the production chain-spec. It's hardcoded JSON and **can't** be changed after the chain launch (I mean the litmus.json), otherwise you get genesis mismatch when starting the nodes.